dawg_python/wrapper.py

```diff
@@ -17,29 +17,29 @@ def __init__(self):
     "Root index"
 
     def has_value(self, index):
-        "Checks if a given index is related to the end of a key."
+        #Checks if a given index is related to the end of a key.
```

Comment:
Why are comments better than docstrings? It is nice to have some docs available at runtime, e.g. in REPL.